src/cascadia/TerminalSettingsModel/KeyChordSerialization.cpp

```diff
@@ -104,14 +105,12 @@ static const std::unordered_map<std::wstring_view, int32_t> vkeyNamePairs {
 // - hstr: the string to parse into a keychord.
 // Return Value:
 // - a newly constructed KeyChord
-KeyChord KeyChordSerialization::FromString(const winrt::hstring& hstr)
+static KeyChord _FromString(const std::wstring_view& wstr)

```

Comment:
unless there's a macro forcing you into it, avoid using underscore-uppercase things